extensions/amp-analytics/0.1/session-manager.js

```diff
@@ -40,6 +40,7 @@ export const SESSION_MAX_AGE_MILLIS = 30 * 60 * 1000;
  */
 export const SESSION_VALUES = {
```

Comment:
This does not apply to this change, so consider it a nit to address in a separate PR.

The rest of the codebase tends to define enum names in `PascalCase` so that you get item references like `SessionValues.SESSION_ID` as opposed to `SESSION_VALUES.SESSION_ID`. Consider changing these enum names and in the rest of `amp-analytics`.